reactive/k8s.py

```diff
@@ -132,24 +139,24 @@ def relation_message():
 
 
 @when('etcd.available', 'tls.server.certificate available')
-@when_not('kubelet.available')
+@when_not('kubelet.available', 'proxy.available')
 def master(etcd):
     '''Install and run the hyperkube container that starts kubernetes-master.
     This actually runs the kubelet, which in turn runs a pod that contains the
     other master components. '''
     render_files(etcd)
-    start_services()
```

Comment:
This actually started the services and we started them again in this method.  Eliminating duplication!